modules/next_stage/test/TestPlan.md

```diff
@@ -1,33 +1,34 @@
+# Next Stage - Test Plan
 1. Create a new candidate and create new timepoint for them.
    [Manual Testing]
-2. Click "Start Visit Stage" on left of instrument_list ensure that page
+2. Click "Start Visit Stage" on left of `instrument_list` ensure that page
    renders.
    [Manual Testing]
-3. Ensure that stage can only be started with the data_entry permission
+3. Ensure that stage can only be started with the `data_entry` permission
    and for candidates at the user's site.
    [Automation Testing]
 4. Ensure that you get an error if dates do not match.
    [Automation Testing]
-5. Ensure that you get an error if scan_done missing
+5. Ensure that you get an error if `scan_done` missing
    [Manual Testing]
 6. Create a timepoint where the age (date entered minus DoB) falls within the
-   AgeMin and AgeMax for a test_battery. Ensure that those instruments are
+   AgeMin and AgeMax for a `test_battery`. Ensure that those instruments are
    inserted (for the correct subprojectID).
    [Manual Testing]
-7. Create a timepoint where there is a test_battery entry for the
-   Visit_label and start the stage. Ensure that those were inserted into
+7. Create a timepoint where there is a `test_battery` entry for the
+   `Visit_label` and start the stage. Ensure that those were inserted into
    the battery, and NOT the ones based on age.
    [Manual Testing]
-8. Ensure that any instruments in test_battery with firstVisit='Y' where
+8. Ensure that any instruments in `test_battery` with firstVisit='Y' where
    inserted if and only if it's the first timepoint started. Ensure that
    firstVisit='N' instruments are inserted if and only if it is NOT the
    first timepoint started, and firstVisit=null whether or not it's the
    first time point.
    [Manual Testing]
 9. Ensure that if a CenterID is set for a row in the test battery, the
-   instrument is inserted for candidates at that centerID (if Visit_label
-   or Age are correct.), and not other centers.
+   instrument is inserted for candidates at that centerID (if `Visit_label`
```

Comment:
instrument is inserted for candidates at that centerID (if instrument `Visit_label` is NULL or identical to the timepoint's one